packages/frontend/src/pages/common/icons/assets/UsdcIcon.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+export function UsdcIcon(props: React.SVGProps<SVGSVGElement>) {
+  return (
+    <svg
+      viewBox="0 0 16 17"
```

Comment:
Why is the viewbox 16x17 here and in other places?

Reply:
That's what figma outputs even though their webapp says dimensions are 16x16 🤷